manifests/init.pp

```diff
@@ -40,16 +40,16 @@
   Stdlib::Absolutepath $config_file,
   String[1] $package_name,
   String[1] $service_name,
-  String[1] $module_prefix,
-  Stdlib::Absolutepath $init_config_file,
-  Hash $init_config_hash,
+  String[0] $module_prefix,
   String[1] $config_file_header,
-  String[1] $package_ensure,
+  Stdlib::Absolutepath $sbin_path,
```

Comment:
This was hard-coded to a location that is wrong on FreeBSD. As this is os-dependant it was moved to Hiera.  Parameter without an explicit default value are expected to be before those with a default one hence:

```diff
-  Stdlib::Absolutepath $sbin_path      = '/usr/sbin',
+  Stdlib::Absolutepath $sbin_path,
```